pkgs/development/python-modules/progressbar2/default.nix

```
@@ -29,17 +29,17 @@ buildPythonPackage rec {
     pytest sphinx flake8 pytestpep8 pytest-flakes pytestcov
     pytestcache freezegun
   ];
-  # ignore tests on the nix wrapped setup.py and don't flake .eggs directory
+  # ignore tests on the nix wrapped setup.py
   checkPhase = ''
     runHook preCheck
-    ${python.interpreter} setup.py test --addopts "--ignore=.eggs"
+    ${python.interpreter} setup.py test
```

they didn't clean cache files before uploading package, that's why some tests are failing
```suggestion
    find . -name '*.py[co]' -delete -o -name __pycache__ -delete
    ${python.interpreter} setup.py test
```

---

made a PR https://github.com/WoLpH/python-progressbar/pull/231